src/utils/proposal-utils.ts

```diff
@@ -30,30 +35,37 @@ export function hasProposalEnded(status, challengeEndDate) {
   )
 }
 
-export function getProposalStatusData(proposal) {
-  const statusData = {}
+export function getProposalStatusData(proposal: {
+  status: string
+  challengeEndDate: Date | any
+}) {
+  let statusData = {}
   if (proposal.status === PROPOSAL_STATUS_EXECUTED_STRING) {
-    statusData.executed = true
+    statusData = { ...statusData, executed: true }
```

Comment:
I think those expansions its not necessary.

typing statusData should be sufficient to keep the old code

Reply:
I am doing this for the sake of `statusData` not losing any data.
.

Reply:
How it will lose data?

Reply:
Whats is the problem of returning a spread object?

Reply:
Whats is the problem of returning a spread object?

Reply:
Want know where it will lose data.

Already told, its not necessary. its verbose, increase code, hard legibility, for what? Nothing.

Reply:
This function itself is all that.

Reply:
I have changed the function

Reply:
I think its improved a lot